dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
I'm generally a fan of what you originally achieved which is having a time-dependent action not require an additional db write at that time, and rather just comparing current timestamp to one that was written to db a while ago.